thirdparty/DiskANN/src/aux_utils.cpp

```diff
@@ -67,7 +67,7 @@ namespace diskann {
     }
 
     size_t num_blocks = DIV_ROUND_UP(fsize, read_blk_size);
-    char  *dump = new char[read_blk_size];
+    char * dump = new char[read_blk_size];
```

Comment:
better to keep this code style un-touched

Reply:
ok, i used diskann clang-format file format it.